src/core/space_interaction_API.jl

```diff
@@ -22,6 +22,7 @@ export move_agent!,
     nearby_agents,
     random_nearby_id,
     random_nearby_agent,
+    random_nearby_position,
```

Comment:
This is an exported function, so it should also be added into the api.md file in the docs. Similarly, it should have a changelog entry in CHANGELOG.md.

Reply:
Thanks for the information
I have added the the function to these files. Nonetheless, I don't know how the addition of a new function changes the number version of the package